src/index.js

```diff
@@ -6,7 +6,7 @@ import GoodByeContext from './GoodByeContext';
 
 export Provider from './GoodByeProvider';
 export withGoodBye from './withGoodBye';
-export useGoodBye from './useGoodBye';
+export useGoodBye from './useGoodbye';
```

Comment:
Perhaps  `useGoodBye` is better than `useGoodbye` in consistence with the others?

Reply:
Well, you do have `withGoodBye` and `GoodByeProvider` just above. So I am consistent ;)

Reply:
Oh you mean filename. Sure, if that's only thing that is bothering you ;)